### PR TITLE
[Magiclyms] Buff Holy Blade and Spiritual Armor spells

### PR DESCRIPTION
**Pull request by Salty-Panda**
#### Summary

SUMMARY: Mods "[Magiclyms] Buff Holy Blade and Spiritual Armor spells"

#### Purpose of change

The Holy Blade spell sucked
Spiritual Armor was marginally useful in some situations

#### Describe the solution

Both have max duration set to 10 mins
Buffed duration increment on level up slightly
Added pure damage type enchantment effect and added 10 pure damage to Holy Sword
Added a few flags to sword summoned via Holy Sword spell: NONCONDUCTIVE, MAGIC_FOCUS and LIGHT_2

#### Describe alternatives you've considered

None

#### Testing

Spawned
Martial art applied properly
Pure damage not listed in the item description

### CODE REVIEW

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 1]
```diff
@@ -743,5 +743,19 @@
     "relic_data": {
       "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "ITEM_DAMAGE_COLD", "add": 6 } ] } ]
     }
+  },
+  {
+    "id": "longsword_holy",
+    "type": "GENERIC",
+    "copy-from": "longsword",
+    "name": { "str": "holy sword" },
+    "description": "This is a summoned medieval sword, in between the lighter arming sword and later two-handed swords in size.  It glows slightly and it's edge is exceedingly sharp.",
```
Comment (LaVeyanFiend) [inline data/mods/Magiclysm/items/ethereal_items.json:752]:
```suggestion
    "description": "This is a summoned medieval sword, between the lighter arming sword and later two-handed swords in size.  It glows slightly and its edge is exceedingly sharp.",
```

Comment (Mrzjadacz) [reply data/mods/Magiclysm/items/ethereal_items.json:752]: > in between the lighter arming sword and later two-handed swords in size

This part was copied from the normal longsword

**Review by KorGgenT — APPROVED** [review]
we need to add a different material than superalloy for magical items. but the buffs look good.

**Comment by actual-nh** [comment]
> we need to add a different material than superalloy for magical items. but the buffs look good.

Possibly Spiritual Armor could eventually be turned into an actual magical item also (with said material substituting for the steel)?

**Pull request merged**